components/eventing-controller/Makefile

```diff
@@ -67,8 +67,27 @@ set-up-local-env:
     export DOMAIN ?= example-domain.com
     export WEBHOOK_TOKEN_ENDPOINT ?= "https://oauth2.${DOMAIN}/oauth2/token"
 
+# Run all tests
 ENVTEST_ASSETS_DIR=$(PROJECT_DIR)/testbin/$(shell uname)
```

Comment:
(nit) Let's move `ENVTEST_ASSETS_DIR=...` before the `setup-envtest` target where it's used.